src/Core/Children.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * @type {Object}
+ */
+var Children = {
+	/**
+	 * @param {*} children 
+	 * @return {Array}
+	 */
+	toArray: function toArray (children) {
+		var array = []
+
+		if (children == null)
+			return array
+		else if (isValidElement(children) || typeof children !== 'object')
+			return [children]
+		else if (children instanceof Array)
+			array = children
+		else if (typeof children.next === 'function')
+			each(children, function (value) {
+				array.push(value)
+			})
+		else if (typeof children[SymbolIterator] === 'function')
+			return this.toArray(child[SymbolIterator]())
+		else
+			return this.toArray([children])
```

Comment:
What triggers this case?

Reply:
It should probably just be `array = [children]` or `return [children]` the object could be a non iterable object like Promise/Date/Error etc.

Reply:
if so, then it seems redundant to have this check

```js
isValidElement(children) || typeof children !== 'object'
```